pkg/sandbox/start.go

```diff
@@ -269,6 +294,13 @@ func StartCluster(ctx context.Context, args []string, sandboxConfig *sandboxCmdC
 		if err := WatchFlyteDeployment(ctx, k8sClient.CoreV1()); err != nil {
 			return err
 		}
+
+		if sandboxConfig.Verbose {
+			if err = getEventLogs(ctx, k8sClient.CoreV1()); err != nil {
+				return err
+			}
+		}
```

Comment:
nit : Can we Stream these while call  watch flyte deployment and also be available once the watch finishes.